Makefile

```diff
@@ -1,6 +1,8 @@
  BAD_FMT_FILES:= $(shell find . -iname '*.go' | grep -v /vendor/ | xargs gofmt -s -l)
  BAD_IMP_FILES:= $(shell goimports -l .)
 
+ EXEC_PATH=/usr/libexec/leapp
```

Comment:
I would suggest here (and also in all others to allow for a PREFIX variable)

Since we will need that from the RPM builds for the buildroot so make install PREFIX=/path/to/buildroot will prefix the location 
Otherwise this looks good to me

Reply:
Good point, THX, and changed.